src/Symfony/Component/Routing/Loader/Configurator/ImportConfigurator.php

```diff
@@ -59,4 +60,18 @@ public function __destruct()
 
         return $this;
     }
+
+    /**
+     * Sets the host to add to use for all child routes.
+     *
+     * @param string|array $host the host, or the localized hosts
+     *
+     * @return $this
+     */
+    final public function host($host): self
+    {
+        $this->addHost($this->route, $host);
```

Comment:
will this work correctly if ppl call `host()` before `prefix()`?

Reply:
Yes, this is what is done in tests.